### DISCUSSION TITLE
Let us Stop Queries in 1 click!

### DISCUSSION THREAD

**Discussion by robarwebservices** [dbeaver/dbeaver#12866] [Q&A]
The default workflow to stop a running query is insane. 3 clicks. 

1. click this: 
 ![image](https://user-images.githubusercontent.com/334314/121206705-137f1800-c84f-11eb-9f24-6f4a702961b4.png)
2. click this:
 ![image](https://user-images.githubusercontent.com/334314/121206838-2bef3280-c84f-11eb-91b7-901b2418e2f6.png)
3. Wait for the Execution Error pop-up???  WHY!  Close it (either ESC or click X)
 ![image](https://user-images.githubusercontent.com/334314/121207010-50e3a580-c84f-11eb-8648-e95f76516dc2.png)

The friction here is INSANE!

Look at SQL Server management studio. Just show a "stop" button somewhere in the header nav bars, let me click it, and when I do, just let it stop the query. NO POP UP! Please!

**Comment by kseniiaguzeeva** [comment]
Could you please disable option **Always run in background** in Preferences>General?
![dbeaver_igXtbxLszN](https://user-images.githubusercontent.com/51405061/122224773-1c11c700-cebd-11eb-8569-64f7b81c1599.png)

![dbeaver_YO25Vdja1T](https://user-images.githubusercontent.com/51405061/122225108-701cab80-cebd-11eb-84c6-8d97c93de6c2.png)

**Comment by robarwebservices** [comment]
It's still so clunky!  Why the need for pop-up dialogs for everything?  Look at how MS Sql Server management studio does it (and MySQL workbench). Run a query, it just starts running... and a little indicator shows near the bottom of the app (no pop up). click 1 button (in the main app) to stop the query. That's it!

**Reply by p91paul** [reply]
I don't know which options you have on, but I can run a query, the progress is shown in the result pane, and I have a cancel button in the middle of it. Also error are shown in the results pane, no popups whatsoever. I believe that to be the default setting...

**Reply by robarwebservices** [reply]
latest version, default settings. It's so painful, I'm ready to ditch it.  (all I do all day is write SQL)
